pkg/cmd/issue/delete/delete_test.go

```diff
@@ -165,7 +165,7 @@ func TestIssueDelete_doesNotExist(t *testing.T) {
 	)
 
 	_, err := runCommand(httpRegistry, true, "13")
-	if err == nil || err.Error() != "GraphQL: Could not resolve to an Issue with the number of 13." {
+	if err == nil || err.Error() != "GQL: Could not resolve to an Issue with the number of 13." {
```

Comment:
Instead of renaming all these error message expectations, how about keeping gh stderr API consistent and rename "GQL" to "GraphQL" in go-gh?